tests/DCPS/ParticipantLocationTopic/permissions_1.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
If you don't absolutely need custom permissions consider using the common permissions files in `test/security/permissions` like the `LargeSample` test does.